### PR TITLE
Update DeveloperGuide.md

### PR DESCRIPTION
**Pull request by jerrelllee**

### CODE REVIEW

**Review by Edsel-Tan — COMMENTED** [review]
LGTM. See feedback.

**Comment by Edsel-Tan** [comment]
LGTM!

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -398,20 +395,22 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
 10. System should have easy-to-use backup and restore functionality, enabling users to create backups of their data, and restore their data in the event of data loss or corruption.
 
 ### Glossary
 
 * **Mainstream OS**: Windows, Linux, Unix, MacOS
 * **Modular**: Codebase is divided into separate parts that can be updated without affecting other parts.
+* **Gracefully recover**: Provides an appropriate solution without crashing the application.
 * **Data retention policy**: Guidelines regarding what data should be kept, how long data should be kept and more.
-* **CLI application**: Any application that takes in input via text, not necessarily through a command console.
+* **CLI - interface**: Any application that mainly takes in input via text, not necessarily through a command console.
```
Comment (Edsel-Tan) [inline docs/DeveloperGuide.md:412]: Perhaps you could emphasize that the interaction with the application is via text?

Comment (jerrelllee) [reply docs/DeveloperGuide.md:412]: Ok! Fixed.

**Pull request merged**